packages/dialtone-vue2/recipes/conversation_view/feed_item_row/feed_item_row.vue

```diff
@@ -271,52 +286,67 @@ export default {
     setHover (bool) {
       this.$emit('hover', bool);
     },
-
-    fade () {
-      // Do not fade if its a default feed row state
-      if (this.state === DEFAULT_FEED_ROW_STATE || this.faded === true) {
-        return;
-      }
-
-      this.$refs.FeedItemRef.$el.classList.remove(FEED_ROW_STATE_BACKGROUND_COLOR[this.state]);
-      this.faded = true;
-    },
   },
 };
 </script>
 
 <style lang="less" scoped>
 .dt-feed-item-row {
-  transition-duration: 2s !important;
+  &--state-searched {
+    background-color: var(--dt-color-surface-warning-subtle);
+  }
+
+  &--state-error {
+    background-color: var(--dt-color-surface-critical-subtle);
+  }
+
+  &--active {
+    background-color: var(--dt-color-surface-secondary-opaque);
+  }
+
+  &--state-transition {
+    transition-duration: 2s;
+    transition-delay: 0s;
+    transition-timing-function: var(--ttf-in-out);
+    transition-property: background-color;
+  }
+
+  width: var(--dt-size-100-percent);
+  box-sizing: border-box;
+  position: relative;
+  padding: var(--dt-space-300) var(--dt-space-500);
+
+  &__content {
+    padding-left: var(--dt-space-300);
+  }
 
   &__header {
     display: flex;
     align-items: center;
+    flex-wrap: wrap;
     gap: var(--dt-space-300);
 
     &__name {
       font-size: var(--dt-font-size-200);
       line-height: var(--dt-font-line-height-300);
       font-weight: var(--dt-font-weight-bold);
-      text-overflow: ellipsis;
-      overflow: hidden;
-      white-space: nowrap;
     }
 
     &__time {
       font-size: var(--dt-font-size-100);
-      margin-top: var(--dt-space-200);
-      line-height: var(--dt-font-line-height-300);
+      line-height: 2.1rem;
```

Comment:
hardcoded line height here since the time uses a different font size than the field it is aligning to, and they need to have the same line height to center align correctly.

Reply:
This will raise a linter warning isn't it?

Reply:
nah only if it's px rather than rem

Reply:
Cool, thought it was a warning on any fixed value instead of CSS var.

Reply:
@braddialpad If the parent is `display: flex`, I'm guessing it is `align-items: center` and can understand your `line-height` workaround. Would `align-items: baseline` work instead?

Reply:
hmm I had some issues with it previously but let me try again after my recent changes.